.changeset/itchy-camels-grin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog': patch
```

Comment:
This changeset line 2 suggests a patch to the catalog plugin - but I don't see any changes for it?

So I'd suggest removal of this line and make this patch as an alpha for the release of the Splunk On-Call plugin itself. (to get it pubilshed to NPM)